prediction_market_agent_tooling/markets/omen/data_models.py

```diff
@@ -36,6 +36,15 @@ def get_boolean_outcome(outcome_str: str) -> bool:
     raise ValueError(f"Outcome `{outcome_str}` is not a valid boolean outcome.")
 
 
+class Condition(BaseModel):
+    id: HexAddress
```

Comment:
conditionId is currently typed as [HexBytes](https://github.com/gnosis/prediction-market-agent-tooling/blob/a62e7808bc0811f7ed814bcf32fb1e8aace3b3d4/prediction_market_agent_tooling/markets/omen/omen_contracts.py#L68), are you sure it should be HexAddress?

```suggestion
    id: HexBytes
```